Woof/Woof/Model/LocalStorage/LocalStorage.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+
+/// A persistent data storage mechanism, allowing you to store key-value pairs
+/// that remain accessible across multiple app launches.
+class LocalStorage {
+    /**
+      Saves a value to the data storage with the specified key.
+
+      - Parameters:
+         - value: The value to be encoded and saved.
+         - key: The key under which the encoded value will be stored.
+     */
+    func save(value: some Encodable, for key: String) {
+        let data = try? JSONEncoder().encode(value)
+        UserDefaults.standard.set(data, forKey: key)
```

Comment:
Could you please explain why do we need the `UserDefaults.standard`?

Reply:
We don't have to create a `UserDefaults.standard` object because it stores the default settings
The best solution would be to create a separate object to store the necessary data using `init(suiteName:)`